components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,178 @@
+$slider__icon-fill-hover         : #007025 !default;
+
+.slider {
+    margin-bottom: $slider__margin;
+    position: relative;
+
+    &__title {
+        margin-bottom: $slider__margin * 2;
+        width: 100%;
+    }
+
+    &__handler {
+        overflow: hidden;
+    }
+
+    &__slides {
+        display: flex;
+    }
+
+    &__item {
+        position: relative;
+        padding: 0;
+        min-width: 50%;
+        padding-right: $slider__item-padding;
+        border: 0;
+
+        @include mq($screen-m) {
+            min-width: 33%;
+        }
+
+        @include mq($screen-l) {
+            min-width: 25%;
+        }
+
+        .product__img-box {
+            min-height: unset;
+        }
+
+        .product__type {
+            padding: 0;
+        }
+    }
+
+    &__link {
+        text-decoration: none;
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__image {
+        margin-bottom: $spacer;
+    }
+
+    &__item-handler {
+        display: flex;
+        flex-wrap: wrap;
+
+        @include mq($screen-m) {
+            flex-wrap: nowrap;
+            justify-content: space-between;
+        }
+    }
+
+    &__name {
+        flex-basis: 100%;
+        font-weight: $slider__name-font-weight;
+
+        @include mq($screen-m) {
+            flex-basis: auto;
+            margin-right: $spacer;
+            margin-bottom: 0;
+        }
+
+        @include mq($screen-l) {
+            font-size: $slider__name-font-size;
+        }
+    }
+
+    &__price {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        color: $slider__price-color;
+        font-weight: $slider__price-font-weight;
+
+        @include mq($screen-m) {
+            flex-flow: column;
+            align-items: flex-end;
+            min-width: fit-content;
+        }
+    }
+
+    &__price-old {
+        display: block;
+        margin-right: $spacer;
+        color: $slider__price-old-color;
+        font-weight: $slider__price-old-font-weight;
+        font-size: $slider__price-old-font-size;
+        text-decoration: line-through;
+
+        @include mq($screen-m) {
+            margin-right: 0;
+        }
+    }
+
+    &__price-new {
+        color: $slider__price-new-color;
+        font-size: $slider__price-new-font-size;
+        font-weight: $slider__price-new-font-wieght;
+    }
+
+    &__badge {
+        position: absolute;
+        top: $spacer;
+        left: 0;
+    }
+
+    &__navigation {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: $spacer;
+
+        @include mq($screen-m) {
+            display: none;
+        }
+    }
+
+    &__navigation-icon {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: $slider__navigation-icon__spacing;
+        width: $slider__navigation-icon__spacing;
+
+        &--left {
+            margin-right: 8px;
```

Comment:
Shouldn't this come form internal var that uses $spacer ?

Reply:
yup, changed to `$slider__navigation-icon-margin`